### DISCUSSION TITLE
Build with scripts

### DISCUSSION THREAD

**Discussion by RavkoDenis** [lxqt/lxqt#2093] [Q&A]
I want to build all LXQt components on my PC (Gentoo) and trying to run scripts from repo, but have some troubles:
1) build_all.sh couldn't find build_all_cmake_projects.sh though it is on the same folder. No problem, started manually.
2) script build_all_cmake_projects.sh contains "sudo" command, but my shell contains "su" only,  No problem, I removed "sudo" from script, moreover, I already under root.
3) Script reached
[100%] Built target sysstat-qt5
...
Install the project...
...
-- Checking for module 'libfm>="1.2.0"'
-- Package 'libfm', required by 'virtual:world', not found
CMake Error at /usr/share/cmake/Modules/FindPkgConfig.cmake:554 (message):
  A required package was not found                                                               
Call Stack (most recent call first):                                                             
  /usr/share/cmake/Modules/FindPkgConfig.cmake:776 (_pkg_check_modules_internal)                 
  CMakeLists.txt:47 (pkg_check_modules)

I expected, that script will install packages in the correct order, moreover, I already use lxqt 0.17 on PC.  
The question is: I forgot to do something, or it is necessary to write issue with full logs?

Thanks in advance.

**Comment by tsujan** [comment]
Another member might answer your question. What follows isn't a reply but a recommendation:

The scripts are there mostly for advanced users or devs who want to test the latest git LXQt in an environment without an LXQt installation. The best and cleanest way of installing git LXQt is by making and installing *installable packages* in the order explained in documentation. Installing through scripts, especially by overwriting your current LXQt installation, could result in a total mess.

**Comment by luis-pereira** [comment]
@RavkoDenis You didn't specify which project failed.
It seems that `libfm` isn't installed.

Personally I don't use the ` build_all_autotools_projects.sh`. I personally don't use it. I use Arch and install `menu-cache`, `libfm-extra`  and `lxmenu-data`  using the package manager. And them just run `build_all_cmake_projects.sh`.
